F3FChrono/data/Round.py

```diff
@@ -33,11 +34,15 @@ def new_round(event, add_initial_group=True):
     def add_group(self, round_group):
         self.groups.append(round_group)
 
-    def handle_terminated_flight(self, competitor, chrono, penalty, valid):
+    def handle_terminated_flight(self, competitor, chronoRun, chronoLap, penalty, valid):
         run = Run()
         run.competitor = competitor
         run.penalty = penalty
-        run.chrono = chrono
+        run.chrono = Chrono()
```

Comment:
I woul have created the Chrono object in the controller but it should work also like this

Reply:
we have two chrono class.
The first in F3FChrono/Chrono.py, and the second one in data/Chrono.py.
I use F3FChrono/Chrono.py in MainUIController and you use data/Chrono in data processing.
Which one should be use ? :)

in F3FChrono/Chrono.py you have a local chrono with time processing. the result time is not the same in GUI. I think change time display in GUI with Chrono.py. This class could handle arduino link.

Reply:
I think these 2 classes don't do exactly the same thing :
* data/Chrono.py is there to store the data only
* F3FChrono/Chrono.py is there to measure the flight time and could take care of the communication with arduino.

I think we need both of them. For more clarity I propose to rename data/Chrono.py to data/RunData.py ... or any name you prefer (FlightData ? RunStatistics ? ... ). I can do the renaming when we are OK on the new name :-)
F3FChrono/Chrono.py should compute the flight statistics and instantiate a data/RunData object to be stored in the database.

Reply:
yes, these make differents things.
I rename F3FChrono/Chrono.py by ChronoHard one hour ago :) and change the code.

Concerning your chrono class in round. Do you make a object for each run ?